storybook/storybook-css/src/components/OrderedList/OrderedList.stories.tsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import { OrderedList } from '@amsterdam/design-system-react/src'
+import type { Meta, StoryObj } from '@storybook/react'
+
+import '@amsterdam/design-system-css/dist/ordered-list/ordered-list.css'
+
+const meta = {
+  title: 'CSS Components/Ordered List',
+  component: OrderedList,
+  args: {
+    children: [
+      <OrderedList.Item key={0}>
+        Dit is het eerste niveau opsommingsteken.
```

Comment:
I’d prefer more realistic content.

[This news article](https://www.amsterdam.nl/nieuws/nieuwsoverzicht/gratis-ov-kinderen/) has a list we can reuse, with longer sentences, too.

To highlight the second level, we can copy a bit of https://nl.wikipedia.org/wiki/Buurten_en_wijken_in_gemeente_Amsterdam.

Reply:
Something like this? I've copied Figma to make the comparison easier.

- Voor deze actie hebben uw kinderen een persoonlijke OV-chipkaart nodig. Hebben zij die nog niet, dan kunt u die nu al aanvragen. Ieder kind heeft een eigen OV-chipkaart nodig.
  - [Bretten West](https://nl.wikipedia.org/wiki/Lange_Bretten)
- U kunt hem aanvragen via [ov-chipkaart.nl](https://www.ov-chipkaart.nl/persoonlijke-ov-chipkaart-aanvragen). De kaart kost € 7,50. U krijgt hem na een dag of 5 thuisgestuurd.
- Op de OV-chipkaart kunt u [gratis reizen voor kinderen aanvragen](https://www.amsterdam.nl/verkeer-vervoer/kinderen-gratis-openbaar-vervoer/) vanaf maandag 3 juli 9.00 uur tot uiterlijk 23 november.
- We helpen mensen die er zelf niet uitkomen. Daarvoor zit een speciaal belteam klaar, bereikbaar via 14 020.
  - Zuidwestkwadrant Osdorp Noord
  - Zuidwestkwadrant Osdorp Zuid
- Het product is geldig in alle bussen, trams en metro’s van GVB. Kinderen reizen met ten minste 1 volwassen begeleider.

Reply:
Great. Just split it into two stories please, mixing two pieces of content makes it make less sense. Also, one example can indicate behaviour for a regular example with multiple lines, the other focuses on the second level.

Reply:
I’ve applied this.